Changelog.md

```diff
@@ -4,6 +4,7 @@ Features:
  * Build System: Update internal dependency of jsoncpp to 1.8.4, which introduces more strictness and reduces memory usage.
  * Code Generator: Use native shift instructions on target Constantinople.
  * Gas Estimator: Only explore paths with higher gas costs. This reduces accuracy but greatly improves the speed of gas estimation.
+ * General: Allow multiple variables to be declared using ``(uint a, uint b) = ...``.
```

Comment:
I think this is misleading. It doesn't allow multiple variable declarations on their own, only declaration with assignment.

Reply:
That's why I put the example there. Any suggestion for a better text?